eslintrc.js

```diff
@@ -1,18 +1,11 @@
 module.exports = {
-  root: true,
-  parser: "babel-eslint",
```

Comment:
I regenerated `.eslintrc.js` based on the latest best practices for Node 16 and ESLint 8.